openvino/itt/include/openvino/macro_overload.hpp

```diff
@@ -0,0 +1,34 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+/**
+ * @brief Macro overloading feature support
+ * @file macro_overload.hpp
+ */
+
+#pragma once
```

Comment:
is it "implementation" file? Should not we add to `EXLUCUDE_PATTERN` for doxygen config?

Reply:
It's internal file which adds macro overloading capabilities. Ok, I'll exclude it from document generation, but in future we can reuse it in other places and, perhepas, it's better to define it in other common library.